.github/workflows/ci.yml

```
@@ -53,11 +59,29 @@ jobs:
           command: bench
           args: --no-run
 
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        features: ["--no-default-features", "--all-features"]
```

```suggestion
        features: ["", "--no-default-features", "--all-features"]
```

If you ever add any non-default features, I'd recommend adding a build with only the default features, but currently it'd be redundant, so take it or leave it.

---

Good call! Even if we don't need it now it seems like it would be easy to forget when we do need it